mission_control/navigator_launch/launch/gnc.launch

```diff
@@ -24,20 +24,9 @@
         <param name="torque_scale" value="500" type="double"/>
     </node>
 
+    <include file="$(find navigator_launch)/launch/subsystems/lqrrt_node.launch"/>
```

Comment:
I told someone this but lqrrt is still in a place where there is some debugging that goes on at lake day and sometimes it needs to be restarted and it's output need to be seen. When you launch with the entire stack the output will be mixed in with the output from everything which makes it hard to work with and restarting isn't easy.